charts/alfresco-repository/values.yaml

```diff
@@ -233,3 +239,7 @@ global:
   alfrescoRegistryPullSecrets: quay-registry-secret
   # -- a fallback for .Values.known_urls that can be shared between charts
   known_urls:
+
+# @ignore
```

Comment:
I think it would just be "doc pollution" as no one is ever supposed to use that. If we end up introducing something valuable to users as tags then we would document it

Reply:
users are not supposed to use that but I see a value telling them explicitly "this is just for testing the chart - don't use it"